src/css.js

```diff
@@ -113,6 +113,18 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableFullscreenBox && window.top !== window.self &&
+		document.msFullscreenElement ) {
```

Comment:
Now I've lost track. What do I need to do to land this fix?
Am 17.06.2015 22:30 schrieb "Michał Gołębiowski" <notifications@github.com>:

> In src/css.js
> <https://github.com/jquery/jquery/pull/2401#discussion_r32670953>:
>
> > @@ -113,6 +113,18 @@ function getWidthOrHeight( elem, name, extra ) {
> >  		styles = getStyles( elem ),
> >  		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
> >
> > +	// Support: IE11+
> > +	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
> > +	if ( !support.reliableFullscreenBox && window.top !== window.self &&
> > +		document.msFullscreenElement ) {
>
> @dmethvin <https://github.com/dmethvin>
>
> I had the same confusion about why msFullscreenElement was being
> (re)checked but then I read the docs and I assume this variable is there
> but null if we're not currently in fullscreen.
>
> That's the usual idiom for DOM stuff. :) A property always exists and is
> just set to null (not undefined) if not defined. The same applies for
> e.g. history.state, you can just check if it's !== undefined to be sure
> it's supported.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jquery/jquery/pull/2401/files#r32670953>.
>